app/src/androidTest/java/com/example/sharingang/EditItemFragmentTest.kt

```diff
@@ -25,14 +27,42 @@ class EditItemFragmentTest {
     private val item = "Test"
     private val editedItem = "Edited"
 
+    @Test
+    fun canEditCategoryOfItem(){
+        onView(withId(R.id.newItemButton)).perform(click());
+        onView(withId(R.id.newItemPrompt)).check(matches(withText("New Item")));
+        onView(withId(R.id.editItemTitle)).perform(
+                typeText("Book_item"),
+                closeSoftKeyboard()
+        );
+
+        onView(withId(R.id.category_spinner)).perform(click());
+        onView(withText("Book")).perform(click());
+
+        onView(withId(R.id.createItemButton)).perform(click());
+
+        onView(withText("Book_item")).check(matches(isDisplayed()));
+        onView(withId(R.id.item_list_view_edit_btn)).check(matches(withText("Edit"))).perform(click());
+
+        onView(withId(R.id.editItemPrompt)).check(matches(withText("Edit Item")));
+        onView(withId(R.id.category_spinner)).check(matches(withSpinnerText("Book")));
+        onView(withId(R.id.category_spinner)).perform(click())
+        onView(withText("Games")).perform(click());
+
+        onView(withId(R.id.editItemButton)).perform(click());
+        onView(withId(R.id.item_list_view_edit_btn)).check(matches(withText("Edit"))).perform(click());
+        onView(withId(R.id.category_spinner)).check(matches(withSpinnerText("Games")));
+    }
+
+
     @Test
     fun anItemCanBeEditedAndSeenOnItemsListFragment() {
         onView(withId(R.id.newItemButton)).perform(click())
         onView(withId(R.id.newItemPrompt))
             .check(matches(withText("New Item")))
         onView(withId(R.id.editItemTitle)).perform(
-            typeText(item),
-            closeSoftKeyboard()
+                typeText(item),
+                closeSoftKeyboard()

```

Comment:
Why the tab?

Reply:
I also find it weird but Android Studio is using by default 8 spaces (why so many wtf) when we split a statement in multiple lines
<img width="238" alt="Screenshot 2021-03-18 at 16 27 43" src="https://user-images.githubusercontent.com/1939395/111651843-dfbfc380-8806-11eb-95d9-3207a1a27941.png">